userspace/libsinsp/base64.h

```diff
@@ -19,6 +19,7 @@
 
 #pragma once
 
+#include <cstdint>
```

Comment:
This change was already merged and can be removed :)  (https://github.com/falcosecurity/libs/pull/1799)

Reply:
I've rebased everything.